public/docs/_examples/cb-item-template/ts/app/main.ts

```diff
@@ -0,0 +1,12 @@
+/* tslint:disable */
```

Comment:
I don't actually know what that line does. I saw that you added it to one of my other cookbooks:

https://github.com/IdeaBlade/angular.io/blame/master/public/docs/_examples/cb-set-document-title/ts/app/main.ts#L1

... so I just copied it over here as well. Should I remove?